regression/automatic_regression.py

```diff
@@ -34,13 +34,13 @@
 modules = [
 
     # ----------------------- Regression List --------------------------
-    'scripts/weights/eVTOL_Weights_Buildup_Regression.py',

```

Comment:
why is this removed?

Reply:
This is temporarily removed. Will be included in the VTOL branch

Reply:
The Weights Branch will pulled in prematurely without the VTOL networks. As a result the quickfix to create the old pull request do not work (without the VTOL branch)